server/internal/testutil/integration.go

```diff
@@ -39,3 +39,11 @@ func IntegrationTest(ctx context.Context, t *testing.T, integrationDeps ...Integ
 		}
 	}
 }
+
+func Must[T any](object T, err error) T {
```

Comment:
What does this function do? Purpose?

Since it's an exposed function, can you add a one line comment that can help people while using it?

```suggestion
// Must ...
func Must[T any](object T, err error) T {
```